plugin/clone/include/clone_common.h

```diff
@@ -17,6 +17,7 @@
 #ifndef CLONE_COMMON_H
 #define CLONE_COMMON_H
 
+#include "clone.h"
 #include "sql/handler.h"
```

Comment:
`#include <string_view>`

Reply:
`std::string_view` for a `get_json_object` parameter

Reply:
Weird I can use string_view without declaring the header still?

Reply:
Probably one of the other headers pulls it in, but
1) transitive includes are brittle as the other headers (including standard library ones) may stop including it at any time
2) source files and headers should include their dependencies directly - this is also something that tooling like include-what-you-use and clangd enforce